script/start-flynn-host

```diff
@@ -112,9 +112,23 @@ main() {
     sudo "${flynn_host}" destroy-volumes ${args[@]}
   fi
 
-  # create IP on lo interface
+  # create IP on enp or lo interface
   info "starting flynn-host using IP ${ip}"
-  sudo ifconfig "lo:${index}" "$ip"
+  local ifname=""
+  for i in $(ls /sys/class/net/);
+  do
+    if [[ $i = "enp"* ]]; then
+      if [[ "$(ifquery $i | grep address)" = "address: 192.0.2.100" ]]; then
+        ifname="$i"
```

Comment:
Does this need a `:$index` suffix? I was unable to get it to work with one, but without only one host will be attached.

Reply:
Just tested again with the index and it works.